safe/definitions/hazard_classifications.py

```diff
@@ -378,7 +378,8 @@
             'color': dark_red,
             'name': tr('Very high'),
             'affected': True,
-            'description': tr('Very High.'),
+            'description': tr('Dry loading on buildings causing structural '
```

Comment:
We usually pull left : 
```
'description': tr(
    'Dry loading ....'
```
Press enter after the bracket.
You will have more space on the line.